README.md

```diff
@@ -23,14 +23,35 @@ up and running to develop and test changes. Follow these steps:
 
 # Build the development image
 
+Clone the repo locally:
+
 ```bash
 git clone https://github.com/talis/talis-php.git
 cd talis-php
-ant build
+```
+
+Manually run a docker build:
+
+```bash
+docker build -t "talis/talis-php" --network=host --build-arg git_oauth_token=<your-git-oauth-token-goes-here> --build-arg persona_oauth_client=<persona-user-goes-here> --build-arg persona_oauth_secret=<persona-secret-goes-here> .
+```
+
+`git_oauth_token` = either generate or use an existing git_oauth token. To generate see guide here: https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line
+
+`persona_oauth_client` = the persona user you want to use.
+
+`persona_oauth_secret` =  the password to the user specified.
+
+Initialise the environment. Run the following command which will download the required libraries.
+
+```bash
+docker-compose run init
```

Comment:
The Dockerfile contains this step. However, it didn't seem to work. The `vendor` directory wasn't present after the docker build had completed, hence, the addition of this manual step.

Reply:
:+1: